src/Overlay/Modal/Modal.styled.ts

```diff
@@ -0,0 +1,81 @@
+import styled, { css, keyframes } from 'styled-components'
+import { Button } from '../../Button'
+
+
+const fadeInAnimation = keyframes`
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+`
+
+export const Dialog = styled.dialog`
+    background-color: var(--md-sys-color-surface-container);
+    border: none;
+    border-radius: var(--border-radius-m);
+    flex-direction: column;
+    gap: 16px;
+    padding: 16px;
+    min-width: 200px;
+    min-height: 100px;
+    max-width: 85%;
+    max-height: 85%;
+   
+
+    /* Backdrop property affects inactive area around modal */
+    &::backdrop {
+        background-color: rgba(0, 0, 0, 0.3);
+    }
+
+    /* Styles for dialogs that carry modal behavior */
+    &:modal {
+       
+    }
+
+    &:modal[open] {
+      animation: ${fadeInAnimation} 0.13s ease-in-out;
+    }
+
+    /* Styles for dialogs that carry non-modal behavior */
+    &:not(:modal) {
+    }
+
+   
+
+`
+
+export const Close = styled(Button)`
```

Comment:
Close button needs to be on the right side not the left

Reply:
Not sure what you ment by that - close button is already on right side... ?

Reply:
Also it needs padding

![image](https://github.com/ynput/ayon-react-components/assets/49156310/a99ae0aa-4274-452e-8a77-08ab1f2dacb1)

Reply:
Ah, you ment bottom one. Sure, changed. But those styles you commented on were related to top right close button

Reply:
Oh sorry my bad, I just read const Close and thought of the "Close" Button.